src/app/page.tsx

```diff
@@ -41,6 +45,8 @@ export default function Home() {
         setArticles(response.articles);
       } catch (err) {
         console.error(err);
+        setIsErrorModalOpen(true);
+        err instanceof AxiosError ? setErrorText(err.message) : 'An error occurred';
```

Comment:
1. These `err.message` error messages don’t make much sense for non-developer users. Please create your own texts for AxiosError
2. What will happen if the err is not an instance of AxiosError? You’ll just put a text inside the block, so it’s basically useless. I guess you wanted to do something like  `setErrorText( err instanceof AxiosError ?err.message  : 'An error occurred');`